runtests.sh

```diff
@@ -118,6 +124,17 @@ fi
 # Unconditionally report on the state of monai
 python -c 'import monai; monai.config.print_config()'
 
+#Cleaning the files when ussing --clean in args, if theres no files exit just exit 
```

Comment:
I think `./runtests.sh --clean` should rm the relevant files and exit, instead of running further tests after rm